travis/script/predicate_functions.sh

```diff
@@ -54,8 +54,8 @@ function is_mri_2plus {
   fi
 }
 
-function is_ruby_23_plus {
-  if ruby -e "exit(RUBY_VERSION.to_f >= 2.3)"; then
+function is_ruby_25_plus {
```

Comment:
Leave the `is_ruby_23_plus` function, and just inline this new check to `rspec_rails_compatible`

Reply:
Ooops, I seem to have missed the "inline" part. Do you want me to fix it real quick?

Reply:
No, all good :)